fractal_tasks_core/create_ome_zarr_multiplex.py

```diff
@@ -266,25 +274,33 @@ def create_ome_zarr_multiplex(
 
         # Identify all wells
         plate_prefix = dict_acquisitions[acquisition]["plate_prefix"]
-        glob_string = f"{image_folder}/{plate_prefix}_*.{image_extension}"
-        logger.info(f"{glob_string=}")
-        plate_image_iter = glob(glob_string)
+        patterns = [f"{plate_prefix}_*.{image_extension}"]
+        if image_glob_patterns:
+            patterns.extend(image_glob_patterns)
+        plate_images = glob_with_multiple_patterns(
+            folder=str(image_folder),
+            patterns=patterns,
+        )
 
         wells = [
-            parse_filename(os.path.basename(fn))["well"]
-            for fn in plate_image_iter
+            parse_filename(os.path.basename(fn))["well"] for fn in plate_images
         ]
         wells = sorted(list(set(wells)))
         logger.info(f"{wells=}")
 
         # Verify that all wells have all channels
         actual_channels = dict_acquisitions[acquisition]["actual_channels"]
         for well in wells:
-            well_image_iter = glob(
-                f"{image_folder}/{plate_prefix}_{well}*.{image_extension}"
+            patterns = [f"{plate_prefix}_{well}*.{image_extension}"]
```

Comment:
See above, would also switch to `f"{plate_prefix}_{well}_*.{image_extension}"`

Reply:
Agreed